### ISSUE TITLE
Confusing the ap_opt()

### ISSUE THREAD

**Issue by rpf5573** [anspress/anspress#472]
ap_opt( 'categoreis_page' ) => 1951
ap_opt( 'categories_page_id' ) => categories

I think this is a little confusing.
And in ap_create_base_page(), 
```
ap_opt( $slug, $page->ID );
ap_opt( $slug . '_id', $page->post_name );
```
I think we should change this code. But, I don't know what is more good way, because ap_opt() is used in may place! It will be more safe for you to change this code.

Check please!

**Comment by rahularyan** [comment]
Yes, you are right. This need to be changed. This function is there in AnsPress since initial release and used in many places.
Need to wait for a major release to fix this. Thanks for reporting.

**Comment by cema93** [comment]
seems fixed?

**Comment by rahularyan** [comment]
Hi @cema93 
No this is still not updated. But it will be updated on next major release which I am already working on. You can track current changes here: https://github.com/anspress/anspress/compare/master...remove-deprecated
